components/collection/drop/GenerativePreview.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="fixed-size border">
     <div
-      class="fixed-top-left border px-4 py-2 bg-background-color has-z-index-1 whitespace-nowrap">
+      class="fixed-top-left border px-4 py-2 bg-background-color z-10 whitespace-nowrap">
```

Comment:
why not `z-[1]` as it's `has-z-index-1` before

Reply:
> why not `z-[1]` as it's `has-z-index-1` before

we are now using tailwind, `has-z-index-1` has now been removed for more consistency

Reply:
yup, I mean why not use `z-[1]` instead of `z-10`

Reply:
`z-10` feels better thant `z-[1]`, it's the only one I've change and it worked well so far

Reply:
Okay